src/dex/generic-rfq/generic-rfq-e2e.test.ts

```diff
@@ -85,7 +85,7 @@ const buildConfigForGenericRFQ = (): RFQConfig => {
   };
 };
 
-const SKIP_TENDERLY = !!getEnv('GENERIC_RFQ_SKIP_TENDERLY', true);
+const SKIP_TENDERLY = getEnv('GENERIC_RFQ_SKIP_TENDERLY', true) === 'true';
```

Comment:
this change will make `SKIP_TENDERLY` `false` in case when `GENERIC_RFQ_SKIP_TENDERLY` is not present in `.env`
we could make the default `true` a string to accommodate

Reply:
updated